lms/djangoapps/lti_provider/urls.py

```diff
@@ -14,5 +14,12 @@
             usage_id=settings.USAGE_ID_PATTERN
         ),
         'lti_provider.views.lti_launch', name="lti_provider_launch"),
-    url(r'^test/?$', 'lti_provider.views.test_launch', name="lti_provider_test_launch"),
+    url(
```

Comment:
It is common functionality for the LTI and Credo Modules. So url must be plased here: https://github.com/CredoReference/edx-platform/blob/integration-ginkgo-qa/lms/urls.py#L256
view also must be placed in the `courseware.views.views` module and must be named without `lti`